themes/bootstrapndlc/templates/posters.html.twig

```diff
@@ -1,5 +1,5 @@
 {% extends 'partials/base.html.twig' %}
-{% set posters = page.collection() %}
+{% set sessionsF = page.collection() %}
```

Comment:
Yay, I fixed it, this is just going to stage for now. Thanks for checking on me!